lib/ocamlorg_web/graphql.mli

```diff
@@ -25,9 +25,13 @@ val get_packages_result
   -> string option
   -> Package.t list
   -> packages_result
+(** This function returns a list of the latest version of all packages *)
+
+val get_single_package : Package.t list -> string -> Package.t option
+(** This function returns the latest version of the pakage name specified *)
```

Comment:
This is just a suggestion (except the typo) but one way that I think leads to better documentation comments is to actually use the function in the comment, e.g.: 
```suggestion
(** [get_single_package packages name] will search the list of packages looking for the package with a name that matches [name], [None] is returned if no package can be found. *)
```

The `[]` in documentation comments will format the docs as code (like backticks in markdown). This is especially useful for functions with labelled arguments.